go/sqltypes/arithmetic_test.go

```diff
@@ -28,6 +28,188 @@ import (
 	"vitess.io/vitess/go/vt/vterrors"
 )
 
+/*
+Test Created for Subtraction function
+func TestSubtraction(t *testing.T) {
+	tcases := []struct {
+		v1, v2 Value
+		out    Value
+		err    error
+	}{{
+
+		//All Nulls
+		v1:  NULL,
+		v2:  NULL,
+		out: NULL,
+	}, {
+
+		// First value null.
+		v1:  NewInt32(1),
+		v2:  NULL,
+		out: NULL,
+	}, {
+
+		// Second value null.
+		v1:  NULL,
+		v2:  NewInt32(1),
+		out: NULL,
+	}, {
+
+		// case with negative value
+		v1:  NewInt64(-1),
+		v2:  NewInt64(-2),
+		out: NewInt64(1),
+	}, {
+
+		// testing for int64 overflow with min negative value
+		v1:  NewInt64(-9223372036854775808),
+		v2:  NewInt64(1),
+		err: vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "BIGINT value is out of range in -9223372036854775808 - 1"),
+	}, {
+
+		v1:  NewUint64(4),
+		v2:  NewInt64(5),
+		err: vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "BIGINT UNSIGNED value is out of range in 4 - 5"),
+	}, {
+
+		v1:  NewUint64(7),
+		v2:  NewInt64(5),
+		out: NewUint64(2),
+	}, {
+
+		v1:  NewUint64(18446744073709551615),
+		v2:  NewInt64(0),
+		out: NewUint64(18446744073709551615),
+	}, {
+
+		// testing for int64 overflow
+		v1:  NewInt64(-9223372036854775808),
+		v2:  NewUint64(0),
+		err: vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "BIGINT UNSIGNED value is out of range in 0 - -9223372036854775808"),
+	}}
+
+	for _, tcase := range tcases {
+
+		got, err := Subtraction(tcase.v1, tcase.v2)
+
+		if !vterrors.Equals(err, tcase.err) {
+			t.Errorf("Subtraction(%v, %v) error: %v, want %v", printValue(tcase.v1), printValue(tcase.v2), vterrors.Print(err), vterrors.Print(tcase.err))
+		}
+		if tcase.err != nil {
+			continue
+		}
+
+		if !reflect.DeepEqual(got, tcase.out) {
+			t.Errorf("Subtraction(%v, %v): %v, want %v", printValue(tcase.v1), printValue(tcase.v2), printValue(got), printValue(tcase.out))
+		}
+	}
+
+}
+*/
```

Comment:
Test cases LGTM, just make sure they are not commented out.

Reply:
I commented it out these test cases because I was in the process of implementing the subtraction function and I wasn't quite done with it, so I commented out parts related to the subtraction function. From next time, I will keep in mind to not comment out test cases.